### ISSUE TITLE
[BUG] Chinese translatation problem

### ISSUE THREAD

**Issue by 12345-mcpython** [NovaMachina/ExNihiloSequentia#436]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Some item don't have chinese traslatation.

![image](https://github.com/NovaMachina-Mods/ExNihiloSequentia/assets/63326494/fc94fdaa-56ec-42d9-9032-5ea45f47ad84)

### Expected Behavior

Have chinese traslation

### Steps to Reproduce

1. switch language to chinese.
2. open item group

### Minecraft Version

1.18.2

### Forge Version

40.2.0

### Ex Nihilo: Sequentia Version

20230528-221633

### Additional Context

wrong item id.

![image](https://github.com/NovaMachina-Mods/ExNihiloSequentia/assets/63326494/b3cbd941-26d8-4667-a6c4-e04b3d24d73e)

![image](https://github.com/NovaMachina-Mods/ExNihiloSequentia/assets/63326494/068926cf-a998-487b-a7c5-5b5e533f2b5a)

### Log File

[latest.log](https://github.com/NovaMachina-Mods/ExNihiloSequentia/files/12557471/latest.log)

**Comment by remplerus** [comment]
that's true, most translations haven't been updated. Feel free to update it with a new PR.
